decidim-core/spec/cells/decidim/notification_cell_spec.rb

```diff
@@ -38,4 +43,26 @@
       expect(subject.to_s).to include("Content moderated")
     end
   end
+
+  context "when action exist" do
```

Comment:
This part reads a bit off when running the spec with format doc: 

```
  when action exist
    Action is present
    and cell does not exist
      Action is present
``` 

I'd prefer to refactor and rename a bit this spec. This is what I have in mind: 

```diff
diff --git a/decidim-core/spec/cells/decidim/notification_cell_spec.rb b/decidim-core/spec/cells/decidim/notification_cell_spec.rb
index b6374b832a..89451babe0 100644
--- a/decidim-core/spec/cells/decidim/notification_cell_spec.rb
+++ b/decidim-core/spec/cells/decidim/notification_cell_spec.rb
@@ -44,23 +44,25 @@ describe Decidim::NotificationCell, type: :cell do
     end
   end

-  context "when action exist" do
-    let(:action_cell) { "decidim/notification_actions/buttons" }
-
+  describe "#action_cell" do
     before do
       allow(notification.event_class_instance).to receive(:action_cell).and_return(action_cell)
       allow(notification.event_class_instance).to receive(:action_data).and_return([{ url: "http://example.com", label: "Some label" }])
     end

-    it "Action is present" do
-      expect(my_cell.action_class).to eq("Decidim::NotificationActions::ButtonsCell")
-      expect(my_cell.action_cell).to eq("decidim/notification_actions/buttons")
+    context "when action cell exists" do
+      let(:action_cell) { "decidim/notification_actions/buttons" }
+
+      it "is present" do
+        expect(my_cell.action_class).to eq("Decidim::NotificationActions::ButtonsCell")
+        expect(my_cell.action_cell).to eq("decidim/notification_actions/buttons")
+      end
     end

-    context "and cell does not exist" do
+    context "when action cell does not exist" do
       let(:action_cell) { "not_existing" }

-      it "Action is present" do
+      it "is not present" do
         expect(my_cell.action_cell).to be_nil
       end
     end
```

And then this reads like this:

```
  #action_cell
    when action cell does not exist
      is not present
    when action cell exists
      is present
```

What do you think? Can you apply it if you like it more? (I can't do it with GH review suggestions as I'll mess the indentation for sure 😄)